Makefile

```diff
@@ -43,6 +43,7 @@ cover: clean
 
 generate: clean
 	$(GO) generate -v ./...
+	$(GO) mod tidy
```

Comment:
The update is in these lines.
https://github.com/cloudflare/circl/blob/master/.github/workflows/ci-actions.yml#L27-L29